Dockerfile.resotobase

```diff
@@ -98,6 +98,12 @@ RUN if [ ${TARGETPLATFORM} = "linux/amd64" ]; then \
         echo "Building for unknown platform - not copying ArangoDB client binaries"; \
     fi
 
+# Download AWS CLI
+WORKDIR /tmp/awscli
+RUN curl "https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip" -o "awscliv2.zip"
```

Comment:
This must work on arm64 and x86_64. See the arango client utilities code above where we check for $TARGETPLATFORM